openwisp_radius/tests/test_api/test_phone_verification.py

```diff
@@ -184,6 +190,30 @@ def test_create_phone_token_400_limit_reached(self):
             else:
                 self.assertEqual(r.status_code, 201)
 
+    @capture_any_output()
+    def test_create_phone_token_400_sms_request_timeout(self):
+        start_time = timezone.now()
+        self._register_user()
+        token = Token.objects.last()
+        token_header = f'Bearer {token.key}'
+        url = reverse('radius:phone_token_create', args=[self.default_org.slug])
+        with freeze_time(start_time):
+            response = self.client.post(url, HTTP_AUTHORIZATION=token_header)
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(
+            response.data['timeout'],
+            self.default_org.radius_settings.get_setting('sms_timeout'),
+        )
+
+        with freeze_time(start_time + timedelta(seconds=15)):
+            response = self.client.post(url, HTTP_AUTHORIZATION=token_header)
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.data['timeout'], 15)
+        self.assertEqual(
+            response.data['non_field_errors'],
+            ['Wait before requesting another SMS token.'],
+        )
+
```

Comment:
Please add another test to ensure the API responds with 201 after the cooldown time.